akka-actor/src/main/scala/akka/actor/dungeon/ChildrenContainer.scala

```diff
@@ -49,13 +50,13 @@ private[akka] object ChildrenContainer {
   final case class Creation() extends SuspendReason with WaitingForChildren
   case object Termination extends SuspendReason
 
-  class ChildRestartsIterable(stats: immutable.MapLike[_, ChildStats, _]) extends PartialImmutableValuesIterable[ChildStats, ChildRestartStats] {
+  class ChildRestartsIterable(stats: ccompat.imm.MapLike[_, ChildStats, _]) extends PartialImmutableValuesIterable[ChildStats, ChildRestartStats] {
```

Comment:
Is this the only place that MapLike is used. This is internal, can we use something else, such as a real Map?